test/functional/developer_portal/admin/messages/outbox_controller_test.rb

```diff
@@ -39,4 +39,22 @@ def test_index
     assert assigned_drop_variables.include?('messages')
     assert assigned_drop_variables.include?('pagination')
   end
+
+  test "Not Creates invalid message" do
+    buyer = FactoryBot.create :buyer_account, :provider_account => @provider
+    post :create, params: { message: { subject: nil, :body => "message with nil subject" }, :to => buyer.id }
+
+    MessageWorker.drain
+    assert msg = Message.last
+    assert_not_includes "message with nil subject", msg.body
+  end
+
+  test "Creates valid message" do
+    buyer = FactoryBot.create :buyer_account, :provider_account => @provider
+    post :create, params: { message: { subject: "Valid Message", :body => "message with subject" }, :to => buyer.id }
+
+    MessageWorker.drain
+    assert msg = Message.last
+    assert "message with subject", msg.body
```

Comment:
this assert with always pass, because your're asserting here that `"message with subject"` is truthy :upside_down_face: